spec/features/users/password_recovery_via_recovery_code_spec.rb

```diff
@@ -24,6 +25,36 @@
     expect(page).to have_content t('headings.account.verified_account')
   end
 
+  scenario 'resets password and reactivates profile with no personal key', email: true, js: true do
+    personal_key_from_pii(user, pii)
+    trigger_reset_password_and_click_email_link(user.email)
+    reset_password_and_sign_back_in(user, new_password)
+    click_submit_default
+    enter_correct_otp_code_for_user(user)
```

Comment:
Should we add a `expect(current_path).to eq reactivate_account_path` here since we're visiting another page below and then manually visiting the expected page later?